core/deployment/src/main/java/io/quarkus/deployment/builditem/DevServicesLocalNetworkBuildItem.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkus.deployment.builditem;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * A marker build item that, if any instances are provided during the build, the containers started by DevServices
+ * will be accessible from Dockers host network. This is mainly used by {@see DevServicesKafkaProcessor} where the
+ * container may need to be accessible from Dockers host network and Dockers internal network.
+ */
+public final class DevServicesLocalNetworkBuildItem extends MultiBuildItem {
```

Comment:
IIUC, the intent of this is to make services available to the docker host, right? If so, I think the name of the build item is misleading - as is the boolean flag that is associated with it (for example in the Kafka dev service you updated)

Reply:
Indeed. Correct... `DevServicesDockerHostNetworkBuildItem` any better?

Reply:
I think some like `DevServicesExposeOnDockerHostBuildItem` captures the intent better. WDYT?

Reply:
@geoand Updated.. 

When you look at this PR one thought I had, whilst writing the JavaDoc, was that instead of a new `BuildItem` I could add "expose to Docker host network" as a parameter to the existing `DevServicesSharedNetworkBuildItem` since the explicit Docker host exposure is only needed when shared network is being used. Then I can examine the `List<DevServicesSharedNetworkBuildItem>` for any member(s) that have the flag set to `true`. 

Too many choices. Life should be easier :laughing:

Reply:
> Too many choices. Life should be easier 

Hahaha.

> add "expose to Docker host network" as a parameter to the existing DevServicesSharedNetworkBuildItem

That's probably a cleaner solution, so +1

Reply:
POOF! It has gone.